src/probe/readiness.ts

```diff
@@ -0,0 +1,27 @@
+import { HealthCheckError } from '@godaddy/terminus';
+import { inject, singleton } from 'tsyringe';
+import { Logger } from '@map-colonies/js-logger';
+import { Services } from '../common/constants';
+
+@singleton()
+export class Readiness {
+  public isReady = true;
+  public constructor(@inject(Services.LOGGER) private readonly logger: Logger) {
+    this.logger = logger;
+  }
+
+  public probe = async (): Promise<void> => {
+    this.logger.debug(`readiness probe is set to ${this.isReady.toString()}.`);
+    if (this.isReady) {
+      return Promise.resolve();
+    } else {
+      // throw new HealthCheckError('Error', 'Readiness has been terminated');
+      throw new HealthCheckError('readiness has been terminated.', []);
```

Comment:
add to common errorr's npm and use it

Reply:
in my opinion, it's not necessary:
our boiler-plate includes the liveness health check with no readiness
this service is the only one who need this use this 'HealthCheckError' from 'terminus'
just because its the only service who has his own liveness and readiness different implementation